openedx/core/djangoapps/schedules/templates/schedules/edx_ace/common/base_body.html

```diff
@@ -80,46 +90,50 @@
-                                                    <a href="https://www.linkedin.com/company/edx">
-                                                        <img src="https://media.sailthru.com/595/1k1/8/o/599f354ec70cb.png"
-                                                             width="32" height="32" alt="edX on LinkedIn"/>
-                                                    </a>
-                                                </td>
-                                                <td height="32" width="42" align="right">
-                                                    <a href="https://www.twitter.com/edXOnline/">
-                                                        <img src="https://media.sailthru.com/595/1k1/8/o/599f354d9c26e.png"
-                                                             width="32" height="32" alt="edX on Twitter"/>
-                                                    </a>
-                                                </td>
-                                                <td height="32" width="42" align="right">
-                                                    <a href="http://www.facebook.com/edX">
-                                                        <img src="https://media.sailthru.com/595/1k1/8/o/599f355052c8e.png"
-                                                             width="32" height="32" alt="edX on Facebook"/>
-                                                    </a>
-                                                </td>
-                                                <td height="32" width="42" align="right">
-                                                    <a href="https://plus.google.com/%2BedXOnline">
-                                                        <img src="https://media.sailthru.com/595/1k1/8/o/599f354fc554a.png"
-                                                             width="32" height="32" alt="edX on Google Plus"/>
-                                                    </a>
-                                                </td>
-                                                <td height="32" width="42" align="right">
-                                                    <a href="https://www.reddit.com/r/edX/">
-                                                        <img src="https://media.sailthru.com/595/1k1/8/o/599f354e326b9.png"
-                                                             width="32" height="32" alt="edX on Reddit"/>
-                                                    </a>
-                                                </td>
+                                                {% if social_media_urls.linkedin %}
+                                                    <td height="32" width="42" align="right">
+                                                        <a href="{{ social_media_urls.linkedin }}">
+                                                            <img src="https://media.sailthru.com/595/1k1/8/o/599f354ec70cb.png"
```

Comment:
I'm surprised we're using sailthru-hosted social media icons...

Reply:
Yeah... open to better ideas.